#float64 confidence
#sensor_msgs/PointCloud2 object_cloud

int32 id
geometry_msgs/Vector3 dimensions
geometry_msgs/PoseStamped pose
